public/js/jquery-extends.js

```diff
@@ -0,0 +1,10 @@
+/**/
+$.fn.extend({
+	animateCss: function (animationName) {
+		var animationEnd = 'webkitAnimationEnd mozAnimationEnd MSAnimationEnd oanimationend animationend';
+		this.addClass('animated ' + animationName).one(animationEnd, function() {
```

Comment:
the `animated` class is undefined in the DOM when no animation is selected by the user.
consider changing to: 
```JS
this.addClass(animationName ? `animated ${animationName}` : '')
```